cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -31,10 +67,31 @@ public boolean install(boolean silent) {
   }
 
   @Override
-  public void installPlugin(PluginDescriptor plugin) {
+  protected void postInstall() {
+
+    super.postInstall();
+    if (this.context.getSystemInfo().isMac()) {
+      setMacOsFilePermissions(getToolPath().resolve("IntelliJ IDEA CE.app").resolve("Contents").resolve("MacOS").resolve(IDEA));
+    }
+  }
 
-    // TODO Auto-generated method stub
+  private static void setMacOsFilePermissions(Path binaryFile) {
 
+    if (Files.exists(binaryFile)) {
+      String permissionStr = FileAccessImpl.generatePermissionString(493);
+      Set<PosixFilePermission> permissions = PosixFilePermissions.fromString(permissionStr);
+      try {
+        Files.setPosixFilePermissions(binaryFile, permissions);
+      } catch (IOException e) {
+        throw new RuntimeException(e);
+      } finally {
+        return;
```

Comment:
This does not seem to add any value and is IMHO only causing confusion.

```suggestion
```

Reply:
Adjusted.